testsuite/features/step_definitions/command_steps.rb

```diff
@@ -559,7 +559,7 @@
 end
 
 # Packages management
```

Comment:
```suggestion
# Repository management
```

Reply:
Right. `# Repositories and packages management`, as the other steps further below are about packages, not about repositories.